<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificationContactWindow</class>
 <widget class="QWidget" name="ModificationContactWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier un contact</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choisissez le contact a modifier :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboContacts"/>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nomLabel">
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nomLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="prenomLabel">
         <property name="text">
          <string>Prénom</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="prenomLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="entrepriseLabel">
         <property name="text">
          <string>Entreprise</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="entrepriseLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="telLabel">
         <property name="text">
          <string>Téléphone</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="telLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="mailLabel">
         <property name="text">
          <string>Mail</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="mailLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="photoLabel">
         <property name="text">
          <string>Photo</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="photoLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="validButton">
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitButton">
         <property name="text">
          <string>Quitter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
